### PR TITLE
Feature/second scoring model

### PR DESCRIPTION
**Pull request by tavurth**

### CODE REVIEW

File: neurons/validator/scoring/models/rewards/enhanced_clip/utils.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    raise ValueError("Unexpected response structure from API")
+
+
+async def openai_breakdown(prompt: str) -> PromptBreakdown:
+    client: AsyncOpenAI = get_openai_client()
+    messages = get_query_messages(prompt)
+    tool = get_prompt_breakdown_function()
+
+    response = await client.chat.completions.create(
+        model="gpt-4",
```
Comment (1ex) [inline neurons/validator/scoring/models/rewards/enhanced_clip/utils.py]: I would suggest switching to gpt-4o, its few times cheaper and for sure will handle the task

File: neurons/validator/scoring/models/rewards/enhanced_clip/utils.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        "messages": [{"role": m.role, "content": m.content} for m in messages],
+        "temperature": 0,
+        "tools": [tool.dict()],
+        "tool_choice": {
+            "type": "function",
+            "function": {"name": tool.function.name},
+        },
+    }
+
+    async with aiohttp.ClientSession() as session:
```
Comment (1ex) [inline neurons/validator/scoring/models/rewards/enhanced_clip/utils.py]: We have httpx (backend client) and aiohttp, maybe should stick to httpx?

File: neurons/validator/scoring/models/rewards/enhanced_clip/utils.py (hunk 1 above)
Comment (tavurth) [reply neurons/validator/scoring/models/rewards/enhanced_clip/utils.py]: Yep good catch!

File: neurons/validator/scoring/models/rewards/enhanced_clip/utils.py (hunk 2 above)
Comment (tavurth) [reply neurons/validator/scoring/models/rewards/enhanced_clip/utils.py]: Yep will update this when I'm back

**Pull request merged**